pkg/lib/page.scss

```diff
@@ -2,6 +2,12 @@ a {
     cursor: pointer;
 }
 
+p + p {
```

Comment:
Of course this is fine as a change but is it really a re-adding? I searched in patternfly repository there does not seem to be any rule that adds margin in the paragraphs.

Reply:
Yes, it is a re-adding.

This is what browsers do by default, when it's not tampered with.